scripts/base/packet-protocols/udp/main.zeek

```diff
@@ -1,5 +1,8 @@
 module PacketAnalyzer::UDP;
 
-#event zeek_init() &priority=20
-#	{
-#	}
+const gre_ports = { 4754/udp } &redef;
```

Comment:
I considered it being in gre instead of UDP, and opted to follow what we did with the ip analyzer. I agree though that it should follow what the other tunnels are doing. We really ought to standardize one of of the two (either parents register children or children register against parents).